openupgrade_scripts/apriori.py

```diff
@@ -22,6 +22,7 @@
     "sale_coupon_commercial_partner_applicability": "coupon_commercial_partner_applicability",
     "sale_coupon_mass_mailing": "coupon_mass_mailing",
     "sale_coupon_portal": "coupon_portal",
+    "sale_coupon_portal_commercial_partner_applicability": "sale_coupon_portal_commercial_partner_applicability",  # noqa: B950
```

Comment:
:slightly_smiling_face:  :point_down: 
```suggestion
    "sale_coupon_portal_commercial_partner_applicability": "coupon_portal_commercial_partner_applicability",  # noqa: B950
```

Reply:
I'm going to try but I don't think it will allow it because when you run pre-commit it reformats it in one line.

Reply:
That's not the fix. Take a closer look :roll_eyes:

Reply:
But if you include the comment "# noqa: B950" pre-commit will still ignore it in a single line.

Reply:
My proposal was about the renaming not the pre-commit

Reply:
ok, I'm still asleep! I had corrected it but I thought it was for pre-commit, sorry